README.md

```diff
@@ -1 +1,58 @@
-# Readability_scorer
+# Readability Scorer
+
+This project is a Readability Scorer that assesses the readability of a given text based on various readability formulas. It provides insights into how easy or difficult it is to comprehend the text. The project is implemented in C and offers a command-line interface for scoring text files.
+
+## Table of Contents
+- [Features](#features)
+- [Getting Started](#getting-started)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Features
+
+- Calculate readability scores using different formulas such as SMOG (Simple Measure of Gobbledygook).
+- Command-line interface for easy text analysis.
+- Input text from files or standard input.
+- Lightweight and fast implementation in C.
+
+## Getting Started
+
+To get started with the Readability Scorer, follow these steps:
+
+1. Clone the repository:
+git clone https://github.com/okeowo-cloud/Readability_scorer.git
+
+2. Build the project using CMake:
+cd Readability_scorer
+cmake .
+make
+
+3. Run the program to analyze a text file.
+./run.sh example.txt
```

Comment:
chmod a+x run.sh
./run.sh example.txt

Reply:
In general README looks good. You didn't mention nothing about running the project in Docker. We have a Dockerfile at the base of the project that can be used to build an image for our application. Your README needs to address how users may also run our project using Docker.

Reply:
You may wrap code in code blocks using three (`) e.g "``` some code ```" to make the readme look more attractive.